hiring-developers-with-readthedocs.rst

```diff
@@ -0,0 +1,79 @@
+.. post:: November 1, 2018
+   :tags: advertising, sustainability, hiring
+   :author: David
+   :location: SAN
+
+
+Tips to Hire Developers with Read the Docs
+==========================================
+
+Read the Docs is probably not the first place you think of if you are recruiting.
+However, over **7 million unique developers use Read the Docs each month** from all over the world.
+We didn't set out to build a better job board,
+but after a number of advertisers used our `ethical ads`_ for recruiting,
+we discovered that Read the Docs was a great place to find developer talent.
+
+.. _ethical ads: https://docs.readthedocs.io/en/latest/advertising/ethical-advertising.html
+
+
+Hiring with Read the Docs works 
+-------------------------------
+
+Developers are not always actively job seeking
+by browsing job boards or company careers pages.
+However, they are on Read the Docs reading about the libraries and frameworks they use.
+Even when people aren't actively looking for a new job,
+many are open to exploring new opportunities.
+We found that developers engage with hiring ads at a higher rate than any other kind of advertising.
+
+.. figure:: img/2018-ctr-adtype.png
+   :alt: Click-through rate by ad type Q3 2018
+   :width: 100%
```

Comment:
`Other` here is a bit funny, as it's among the highest ranking and isn't clear what that means. I'd maybe remove it?

Reply:
Also it's probably useful to sort them by % or something to make it a bit easier to process.

Reply:
is it possible to sort increasing instead of decreasing? just by looking at it (without reading the numbers and the meaning) seems like things are going down :)

It's more a psychological thing, I guess.